paket-files/haf/YoLo/YoLo.fs

```diff
@@ -622,7 +622,7 @@ module App =
 
   /// Get the assembly resource
   let resource name =
-#if DNXCORE50
+#if NETSTANDARD1_5
```

Comment:
oh I see, I'll need to understand whether `netstandard` is going to replace `dnxcore` completely before applying this change